BB-SC/MWLayer/Communication/BB_SecOS.md

```diff
@@ -0,0 +1,77 @@
+
+# SecOS
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Communication
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+MWLayer
+
+## Known Implementation
+
+## ID (unique name)
+
+## Description
+<!-- General Description of the BB -->
+Define standard service for secured onboard communication
```

Comment:
Unclear what specifically this BB is supposed to describe. TLS? SecOC?

Reply:
Agreed. Needs further described. -> WP2

Reply:
add new field: "Expert-discussion", status: "in discussion"

Reply:
done